noggin/tests/unit/conftest.py

```diff
@@ -102,27 +100,27 @@ def ipa_testing_config(vcr_session):
 def make_user(ipa_testing_config):
     created_users = []
 
-    def _make_user(name):
+    def _make_user(a_uid):
```

Comment:
same comment here as before. you can leave the function the same but just assign `a_uid` if you like.